File: week3/Cho-Young-Il-2.java
import java.util.Scanner;
 
public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int sum = 0;
        int count = 2;
        while(count > 0) {
            sum += in.nextInt();
            count--;
        }
        System.out.println(sum);
    }
}

File: homework/week4/Cho-Young-Il-2.js
import java.util.Scanner;
 
public class Main{
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = Integer.parseInt(in.nextLine());
        int[] fibo = new int[n + 1];
        fibo[0] = 0;
        fibo[1] = 1;
         
        for(int i = 2; i <= n; i++) {
            fibo[i] = fibo[i - 1] + fibo[i - 2];
        }
         
        System.out.println(fibo[n]);
    }
}